amazoncorretto-11/Dockerfile.windows

```diff
@@ -0,0 +1,46 @@
+# escape=`
```

Comment:
For lack of highlighting, see https://github.com/github/linguist/issues/4566#issuecomment-515741646.